Docs 6872 pm2.x delete certs m (#74)
mulesoft/docs-partner-manager: modules/ROOT/pages/configure-host.adoc

. Click *New* within the *Certificates* section to begin to upload any needed digital certificates (such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol).

> . Click *Choose file* to select the certificate (typically a `.PEM` file), and click *Upload certificate* to upload your chosen certificate file. See xref:Certificates.adoc[Certificates] for more info on configuring certificates in Partner Manager.

Reviewer: ```suggestion
. Click *Choose file* to select the certificate (typically a `.PEM` file), and click *Upload certificate* to upload your chosen certificate file. 
See xref:Certificates.adoc[Certificates] for more information about configuring certificates in Partner Manager.
```

Author: Fixed.

Author: Fixed.

+

> NOTE: If a single certificate is uploaded, Partner Manager prevents you from deleting your only certificate.

Reviewer: Does this really belong at this location? Is the user deleting or planning to delete a certificate at this point?

Author: Fixed.

+
. Click *New* from within _Endpoints: Receive from partners_ or _Endpoints: Targets at <partner name>_ for the Host, and select your protocol type.

. Set up your endpoint based on the protocol type you selected.
+

> For more information on your protocol type refer to:

Reviewer: ```suggestion
For more information about your protocol type, refer to:
```

Author: Fixed.

* xref:endpoint-sftp-receive-target.adoc[SFTP Receive Endpoint Settings]

* xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
+

> . Save when you are done.

Reviewer: How? Do they click something?

Author: Fixed.

> . If you need to change any of the information that was completed when the payload storage was set up, click *Edit* within the *Payload storage* section.

Reviewer: Could this be "If you need to make changes later, click...."

Author: Fixed.

> For example, URL location, timeout settings, and acknowledgments.

Reviewer: This is a fragment. Please incorporate it into the sentence above or make it complete in itself.

Author: Fixed.

Refer to xref:x12-identity-settings.adoc[Identifiers] for information on deleting an identifier.

> Refer to xref:endpoints.adoc[Endpoint Types and Protocols] for information on deleting an endpoint.

Reviewer: GLOBALLY change "information on" to "information about" in all writing.
I will stop marking it from here on.

Author: Fixed.
